### PR TITLE
Fix/write coin basis

### LINKED ISSUE #465: Sensitivity analysis bug when tested with windows
Following error message, when testing the functionality on the studies of the "example" folder :

image

An idea would be to add systematically tests with these studies of the "example" folder on windows before each release.

### PR DESCRIPTION
**Pull request by tbittar**
Fix Cbc solver interface basis was not written/read correctly to enable warm start, closes #465

### CODE REVIEW

File: src/cpp/multisolver_interface/SolverXpress.cpp [hunk 1]
```diff
@@ -119,7 +119,7 @@ void SolverXpress::write_prob_lp(const std::filesystem::path &filename) {
 }
 
 void SolverXpress::write_basis(const std::filesystem::path &filename) {
-  int status = XPRSwritebasis(_xprs, filename.string().c_str(), "");
+  int status = XPRSwritebasis(_xprs, filename.string().c_str(), "n");
```
Comment (JasonMarechal25) [inline src/cpp/multisolver_interface/SolverXpress.cpp]: Can you extract the magic value "n" in a named constant please ?

File: src/cpp/sensitivity/AnalysisFunctions.cpp [hunk 2]
```diff
@@ -31,21 +27,29 @@ SolverAbstract::Ptr get_sensitivity_problem(
...
+      sensitivity_problem = pb_modifier.changeProblem(nb_candidates);
+      break;
     }
 
     default:
       std::cerr << "Unrecognized Sensitivity Problem type" << std::endl;
       return nullptr;
   }
+
+  if (std::ifstream basis_file(input_data.basis_file_path); basis_file.good()) {
```
Comment (JasonMarechal25) [inline src/cpp/sensitivity/AnalysisFunctions.cpp]: If the file is not good nothing happens. Should we at least warn users or developers with log ?

**Review by JasonMarechal25 — CHANGES_REQUESTED** [review]

File: src/cpp/sensitivity/AnalysisFunctions.cpp (hunk 2 above)
Comment (tbittar) [reply src/cpp/sensitivity/AnalysisFunctions.cpp]: Yes, we can add a warning

File: src/cpp/multisolver_interface/SolverXpress.cpp (hunk 1 above)
Comment (a-zakir) [reply src/cpp/multisolver_interface/SolverXpress.cpp]: in doc it's a `const char* flags `

**Review by a-zakir — APPROVED** [review]

**Pull request merged**